src/components/Translatathon/TranslatathonPrizes.tsx

```diff
@@ -0,0 +1,88 @@
+import Emoji from "@/components/Emoji"
+
+const TranslatathonPrizes = () => {
+  return (
+    <div className="flex w-full flex-col gap-8 rounded-lg">
+      <div className="m-0 py-4 sm:m-auto">
+        <div className="flex flex-row gap-4">
+          <div className="flex h-32 w-32 items-center justify-center rounded-lg p-1 shadow-md">
+            <div className="h-16 w-16 items-center justify-center">
+              <Emoji text="🥇" fontSize={64} />
+            </div>
+          </div>
+          <div className="flex flex-col items-center justify-center">
+            <p className="text-2xl font-bold">1st place</p>
+            <p className="text-4xl font-bold">5000$</p>
```

Comment:
If we don't want to use the Intl library to format this, I would use the dollar sign in front (throughout):

```suggestion
            <p className="text-4xl font-bold">$5000</p>
```